client/src/main.ts

```diff
@@ -105,8 +105,9 @@ async function setupApp(): Promise<void> {
     window.isDev = (): boolean => false;
     window.isLinux = (): boolean => isLinux;
 
-    if (configPath) {
-      window.getConfigDir = (): string => configPath;
+    if (testbedDiscriminantPath) {
+      window.getConfigDir = (): string => testbedDiscriminantPath;
+      window.getDataBaseDir = (): string => testbedDiscriminantPath;
```

Comment:
@max, why those attribute are not constant ? because you want to be absolutely sure that are not overwritten ?